docs/CONTRIBUTING.md

```diff
@@ -24,19 +30,57 @@ If you are familiar with contributing to open source software, you can safely sk
 
 ## Getting Started<a name="getting-started"></a>
 
-To begin contributing to HJR, you will first need to create a fork off of the main branch in order to make changes. For info on how to do this, we recommend GitHub's own [documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).
+To begin contributing to HJR, you will first need to create a fork off of the
+main branch in order to make changes. For info on how to do this, we recommend
+GitHub's own
+[documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).
 
-Once you have your own fork, it is recommend that you create a branch for the changes or additions you seek to make, to make it easier to set up multiple PRs from your fork. To do so, you can read this [article](https://www.geeksforgeeks.org/how-to-create-a-new-branch-in-git/) that will also explain branches for you. Don't worry too much about the technical details, the most important thing is to make and switch to a branch from HEAD.
+Once you have your own fork, it is recommend that you create a branch for the
+changes or additions you seek to make, to make it easier to set up multiple PRs
+from your fork. To do so, you can read this
+[article](https://www.geeksforgeeks.org/how-to-create-a-new-branch-in-git/) that
+will also explain branches for you. Don't worry too much about the technical
+details, the most important thing is to make and switch to a branch from HEAD.
 
-You can now make your changes in your editor of choice. After committing your changes, you can run:
+### Commit format<a name="commit-format"></a>
+
+For consistency, we do enforce a strict (but simple) commit style, that will be
+linted against. The format is as follows:
+
+```console
+<top_level_scope>/[/<specific_scope>]: <message>
+
+<body>
+```
+
+- \<top_level_scope>: the top-level scope of your commit (e.g. if making a
+  change to a program module, this would be `programs`).
+
+- \[/\<specific_scope>\]: the specific scope of your commit (e.g. if you're
```

Comment:
There is a stray forward-slash.

Reply:
It's not a stray slash, it's because the full scope looks like foo/bar, I wasn't sure where else to put it

Reply:
Ah, then it makes sense